first-analysis-steps/files-from-grid.md

```diff
@@ -5,49 +5,44 @@
 {% endobjectives %} 
 
 In the [previous section](bookkeeping.html), we obtained a file called 
-`MC_2012_27163003_Beam4000GeV2012MagDownNu2.5Pythia8_Sim08e_Digi13_Trig0x409f0045_Reco14a_Stripping20NoPrescalingFlagged_ALLSTREAMS.DST.py` 
+`MC_2016_27163002_Beam6500GeV2016MagDownNu1.625nsPythia8_Sim09b_Trig0x6138160F_Reco16_Turbo03_Stripping28NoPrescalingFlagged_ALLSTREAMS.DST` 
 which contains the following section:
 
 ```python
-IOHelper('ROOT').inputFiles(['LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000001_1.allstreams.dst',
-'LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000002_1.allstreams.dst',
-'LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000003_1.allstreams.dst',
-'LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000004_1.allstreams.dst',
-'LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000005_1.allstreams.dst',
-'LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000006_1.allstreams.dst',
-'LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000007_1.allstreams.dst',
-'LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000008_1.allstreams.dst',
-'LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000009_1.allstreams.dst',
-'LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000010_1.allstreams.dst'
+IOHelper('ROOT').inputFiles(['LFN:/lhcb/MC/2016/ALLSTREAMS.DST/00062514/0000/00062514_00000001_7.AllStreams.dst',
+'LFN:/lhcb/MC/2016/ALLSTREAMS.DST/00062514/0000/00062514_00000002_7.AllStreams.dst',
+'LFN:/lhcb/MC/2016/ALLSTREAMS.DST/00062514/0000/00062514_00000003_7.AllStreams.dst',
+'LFN:/lhcb/MC/2016/ALLSTREAMS.DST/00062514/0000/00062514_00000004_7.AllStreams.dst',
+...
 ], clear=True)
 ```
-which is just a collection of **L**ogical **F**ile **N**ames on the grid.
+which is just a collection of **L**ogical **F**ile **N**ames on the grid. ****
```

Comment:
What does the four-asterisk thing do here?

Reply:
Sorry--these can be removed. Atom, the editor I use, gets confused when displaying the file when characters abut the **. You can actually see it in GitHub's Markdown too--without the ****, the entire remainder of the file gets bolded.

This was just for my own viewing sanity while editing.